### PR TITLE
docs: improve docs

### PR DESCRIPTION
**Pull request by fedeci**
## Description

 #931, #883 and #791 

Changes: 
- New FAQ page
- Warning about not using `notEmpty()` for arrays
- New notice about Standard validators/sanitizers type conversion to string.
- Explanation of how to properly validate/sanitize all array values
- Update validator.js repo URL

## To-do list

- [x] I have added tests for what I changed.

- [x] This pull request is ready to merge.

### CODE REVIEW

**Comment by fedeci** [comment]
@favph what do you think?

**Comment by favph** [comment]
@fedeci looks good to me.

File: docs/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Getting Started
 ---
 
 express-validator is a set of [express.js](http://expressjs.com/) middlewares that wraps
-[validator.js](https://github.com/chriso/validator.js) validator and sanitizer functions.
+[validator.js](https://github.com/validatorjs/validator.js) validator and sanitizer functions.
```
Comment (gustavohenke) [inline docs/index.md:7]: Update on README too?

File: website/sidebars.json [hunk 2]
```diff
@@ -17,6 +17,7 @@
       "sanitization-chain-api",
       "matched-data-api",
       "validation-result-api"
-    ]
+    ],
+    "More": ["faq"]
```
Comment (gustavohenke) [inline website/sidebars.json:21]: Can you also make the migration guide accessible here?
Looks like I completely forgot of it in #948 🤦

**Review by gustavohenke — APPROVED** [review]
Thanks!
I'd love to think of ways to improve the API instead of documenting work arounds -- but for now that's _much_ better than us (you mostly!) replying to issues that pop up all the time...

**Comment by fedeci** [comment]
> Thanks!
> I'd love to think of ways to improve the API instead of documenting work arounds -- but for now that's _much_ better than us (you mostly!) replying to issues that pop up all the time...

Definitely we should elaborate a strategy to resolve all the most critical bugs in `express-validator@7`.

**Pull request merged**